src/Networking/v2/Models/InterfaceAttachment.php

```diff
@@ -0,0 +1,26 @@
+<?php declare(strict_types=1);
+namespace OpenStack\Networking\v2\Models;
+
+use OpenStack\Common\Resource\Listable;
+use OpenStack\Common\Resource\OperatorResource;
+
+/**
+ * @property \OpenStack\Networking\v2\Api $api
+ */
+class InterfaceAttachment extends OperatorResource implements Listable
+{
+    /** @var string */
+    public $id;
+
+    /** @var int */
+    public $device;
+
+    /** @var string */
+    public $serverId;
+
+    /** @var string */
+    public $interfaceId;
```

Comment:
Does this not need aliases? I can't see any tests which verify the HTTP response